packages/rules-unit-testing/package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "@firebase/testing",
-  "version": "0.20.9",
+  "name": "@firebase/rules-unit-testing",
+  "version": "0.30.0",
```

Comment:
Should the new package start at `0.1.0`? Also the changeset will auto bump the minor version during the release, so you want to use a version number that is one minor version smaller.

Reply:
Or should it start at `1.x` to signify that we're breaking old APIs AND stabilizing into new APIs? Just throwing ideas out there

Reply:
I'm cool with `1.x` ... so does that mean I should make it `0.1.0` and then do a major changeset?

Reply:
Going with 1.0

Reply:
Yes